dotnetv3/S3/EnableNotificationsExample/EnableNotificationsExample/EnableNotifications.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+namespace EnableNotificationsExample
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    /// <summary>
+    /// This example shows how to enable notifications for an Amazon Simple
+    /// Storage Service (Amazon S3) bucket. The examples use the AWS SDK for
+    /// .NET 3.7, and .NET 5.0.
+    /// </summary>
+    public class EnableNotifications
+    {
+        /// <summary>
+        /// The Main method initializes the variables and the client object
+        /// and then passes the values to EnableNotificationAsync.
+        /// </summary>
+        public static async Task Main()
+        {
+            const string bucketName = "doc-example-bucket1";
+            const string snsTopic = "arn:aws:sns:us-east-2:709259161347:bucket-notify";
+            const string sqsQueue = "arn:aws:sqs:us-east-2:709259161347:Example_Queue";
+
+            IAmazonS3 client = new AmazonS3Client(Amazon.RegionEndpoint.USEast2);
+            await EnableNotificationAsync(client, bucketName, snsTopic, sqsQueue);
+        }
+
+        /// <summary>
+        /// This method makes the call to the AWS SDK PutBucketNotificationAsync
+        /// method.
+        /// </summary>
+        /// <param name="client">An initialized Amazon S3 client used to call
+        /// the PutBucketNotificationAsync method.</param>
+        /// <param name="bucketName">The name of the bucket for which
+        /// notifications will be turned on.</param>
+        /// <param name="snsTopic">The ARN for the Amazon Simple Notification
+        /// Service topic associated with the S3 bucket.</param>
+        /// <param name="sqsQueue">The ARN of the Amazon Simple Queue Service
```

Comment:
Amazon Simple Queue Service  -> Amazon Simple Queue Service (Amazon SQS)

Reply:
changed to Amazon SQS queue